src/bin/main.rs

```diff
@@ -43,7 +43,7 @@ fn main() -> Result<()> {
     check_if_terminal();
 
     // Read from config file.
-    let config_path = read_config(matches.value_of("config_location"))
+    let config_path = read_config(matches.get_one::<String>("config_location").map(|x| &**x))
```

Comment:
nit: feel free to remove the `.map(|x| &**x)` part and just change the function signature of `read_config` to take an `Option<&String>` if you want.

Reply:
updated